### PR TITLE
bump version to 0.0.4

### PR DESCRIPTION
**Pull request by komidore64**
moving spec from katello-ballpeen

### CODE REVIEW

File: rubygem-hammer_cli_foreman_tasks.spec [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+- fix package (jmontleo@redhat.com)
+
+* Thu Feb 27 2014 Jason Montleon <jmontleo@redhat.com> 0.0.1-2
+- add unpackaged files to package (jmontleo@redhat.com)
+- fix Require for hammer_cli_foreman instead of hammer_cli
+  (jmontleo@redhat.com)
+
+* Thu Feb 27 2014 Jason Montleon <jmontleo@redhat.com> 0.0.1-1
+- new package built with tito
+
```
Comment (jlsherrill) [inline rubygem-hammer_cli_foreman_tasks.spec]: I'd purge all these changelog entries

Comment (komidore64) [inline rubygem-hammer_cli_foreman_tasks.spec]: the "new package built with tito" messages? or all?

Comment (jlsherrill) [inline rubygem-hammer_cli_foreman_tasks.spec]: everything but 1 'new package built with tito'.  There's a lot of 'downstream' comments.

Comment (komidore64) [inline rubygem-hammer_cli_foreman_tasks.spec]: fair. removing all but the top.

**Comment by komidore64** [comment]
@jlsherrill changelog trimmed.

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**